### ISSUE TITLE
Recreate in more languages/frameworks

### ISSUE THREAD

**Issue by t4lz** [metalbear-co/playground#4]
People usually prefer to test out mirrord with an app built with the same language and framework as they use.

Let's create versions of the playground in many popular languages/frameworks.
I suggest we keep them here in the same repo, each language-framework combination (including go) in its own directory.

- [ ] Java - Spring.
- [ ] C# - asp.NET
- [ ] Python - ?

Please feel free to edit the list. Let's convert the frameworks into their own issues when they are prioritized.
Also, if users add requests for frameworks in separate issues, let's add them to this list.

**Comment by t4lz** [comment]
Closing in favour of https://github.com/metalbear-co/mirrord.dev/issues/62

**Comment by t4lz** [comment]
I misunderstood, the playground **is** going to be used by users to try out mirrord, so let's add more languages.
(But only after we finish the other issues and use this repo in the tutorial on the website.)
